src/sql/workbench/contrib/objectExplorer/browser/serverTreeView.ts

```diff
@@ -46,7 +46,8 @@ export class ServerTreeView extends Disposable {
 	private _buttonSection: HTMLElement;
 	private _treeSelectionHandler: TreeSelectionHandler;
 	private _activeConnectionsFilterAction: ActiveConnectionsFilterAction;
-	private _tree: ITree;
+	// Public set for testing only - this should not be set directly
+	public tree: ITree;
```

Comment:
Im not a fan of making things public for testing only. It tells me the code isn't written correctly and it breaks the meaning of what something `public` is.

Reply:
Sure, any better suggestions? I guess I could keep it private and set it directly on the object like the old test code was doing - would that be acceptable?

Reply:
Actually I guess I should be able to mock it to just return the tree when called too. Let me know if you have other better suggestions.